Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -24,6 +24,8 @@ export interface Spec extends TurboModule {
       +tintColor?: ?number,
       +userInterfaceStyle?: ?string,
       +disabledButtonIndices?: Array<number>,
+      +icons?: ?Array<Object>, // TODO: This should be a ResolvedAssetSource
+      +tintIcons?: ?boolean,
```

Comment:
i think it's more clear to make this a verb, i.e. `shouldTintIcons`. otherwise, i would think this is an array of icon metadata, and since we lose the type in the obj-c layer, that could be very misleading.

also, this conforms more to obj-c naming standards